packages/components/src/components/action/action.scss

```diff
@@ -63,27 +61,22 @@
   --calcite-internal-action-padding-inline: var(--calcite-spacing-xxs);
 }
 
-.button {
-  border-radius: inherit;
-}
-
-@mixin action-indicator() {
+@mixin action-indicator($offset: -0.275rem) {
   position: relative;
   &::after {
     content: "";
     @apply absolute
       h-2
       w-2
       rounded-full;
-    inset-block-end: calc(-0.275rem);
-    inset-inline-end: calc(-0.275rem);
+    inset-block-end: calc($offset);
```

Comment:
Side note: `calc` doesn't seem to be necessary here.

Reply:
Good catch, I completely overlooked that :)